tslint.json

```diff
@@ -68,12 +68,14 @@
     "variable-name": false,
     "whitespace": [true, "check-branch", "check-decl", "check-operator", "check-separator", "check-type"],
     "no-output-on-prefix": true,
-    "directive-selector": [true, "attribute", "sfeir", "camelCase"],
+    "directive-selector": [true, "attribute", "sfeir", "camelCase", "kebab-case"],
```

Comment:
I think it's a bad practice. Why did you add that? If it's because of an existing use case, I think fixing it instead of changing the rules would be better.